app/cdap/components/GridTable/components/GridKPICell/index.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Card, TableCell, Typography } from '@material-ui/core';
+import React from 'react';
+import { useGridKPICellStyles } from './styles';
+import { IGridKPICellProps } from './types';
+
+export const GridKPICell: React.FC<IGridKPICellProps> = ({ metricData }) => {
+  const classes = useGridKPICellStyles();
+
+  const { values } = metricData;
+
+  return (
+    <TableCell className={classes.tableHeaderCell}>
+      <Card className={classes.root} variant="outlined">
+        {values &&
+          Array.isArray(values) &&
+          values.length &&
+          values.map((eachValue: { label: string; count: number }) => (
```

Comment:
can we get a better name here? metricValue?

Reply:
or just value

Reply:
Done